backend/tests/slack/commands/management/slack_sync_messages_test.py

```
@@ -0,0 +1,638 @@
+            patch.object(Message, "bulk_save") as mock_bulk_save,
+            patch("builtins.print"),
+        ):
+            mock_message_filter.return_value.order_by.return_value.first.return_value = None
+            mock_update_data.return_value = Mock(spec=Message)
+            command.stdout = stdout
+            command.handle(batch_size=200, delay=0.5, channel_id=None, include_threads=False)
+
+        assert mock_update_data.call_count == CONSTANT_2
+        mock_bulk_save.assert_called_once()
+
+        mock_client.conversations_replies.assert_not_called()
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)